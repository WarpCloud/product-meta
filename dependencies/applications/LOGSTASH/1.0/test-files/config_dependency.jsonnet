{
  application_type: "LOGSTASH",
  application_name: "logstash1",
  application_version: "1.0",
  user_config: {
    use_search: "true",
  },
}
