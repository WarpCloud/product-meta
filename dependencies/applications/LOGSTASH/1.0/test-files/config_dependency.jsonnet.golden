{
   "Customized_Namespace": "default",
   "Develop": false,
   "TCU": {
      "logstash": {
         "c": 0.25,
         "m": 0.5,
         "s": 0.5
      }
   },
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "logstash1",
            "application_type": "LOGSTASH",
            "use_search": "true"
         },
         "configs": {
            "logstash_cpu_limit": 1,
            "logstash_cpu_request": 1,
            "logstash_memory_limit": 2,
            "logstash_memory_request": 2,
            "logstash_tmp_storage": {
               "accessMode": "ReadWriteOnce",
               "limits": {
                  "blkio.throttle.read_iops_device": 500,
                  "blkio.throttle.write_iops_device": 500
               },
               "size": "10Gi",
               "storageClass": "silver"
            }
         },
         "dependencies": [
            {
               "moduleName": "elasticsearch",
               "name": "elasticsearch"
            }
         ],
         "instance_settings": { },
         "moduleName": "logstash",
         "name": "logstash1-logstash",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
